zomes/chat/src/entries/message/handlers.rs

```diff
@@ -209,6 +209,33 @@ fn active_chatters(chatters_path: Path) -> ChatResult<(usize, Vec<AgentPubKey>)>
     Ok((total, active))
 }
 
+pub(crate) fn get_active_chatters() -> ChatResult<ActiveChatters> {
+    let me = agent_info()?.agent_latest_pubkey;
+    let chatters_path: Path = chatters_path();
+    let (_total, mut chatters) = active_chatters(chatters_path)?;
+    chatters.retain(|a| *a != me);
+    Ok(ActiveChatters { chatters })
+}
+
+pub(crate) fn signal_specific_chatters(input: SignalSpecificInput) -> ChatResult<()> {
+    let mut chatters = input.chatters;
```

Comment:
is there a more idiomatic way of doing this in rust than having a mutable vec that might get appended to?

Reply:
this is the most idiomatic way. It's great to mutate the existing vec because you have the potential to reuse the allocation.

Reply:
Thanks!